docs/scos/dev/front-end-development/202304.0/zed/oryx-builder-overview-and-setup.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Oryx builder overview and setup
+description: Oryx is the Spryker projects frontend helper. The purpose of Oryx is to simplify the asset building process, giving developers the freedom to choose and configure the preprocessors for the frontend.
+last_updated: Apr 3, 2023
+template: howto-guide-template
+originalLink: https://documentation.spryker.com/2021080/docs/oryx
+originalArticleId: 466d3f47-f666-4e78-9a16-65afb3d615b7
+redirect_from:
+  - /2021080/docs/oryx
+  - /2021080/docs/en/oryx
+  - /docs/oryx
+  - /docs/en/oryx
+  - /v6/docs/oryx
+  - /v6/docs/en/oryx
+  - /v5/docs/oryx
+  - /v5/docs/en/oryx
+  - /v4/docs/oryx
+  - /v4/docs/en/oryx
+  - /v3/docs/oryx
+  - /v3/docs/en/oryx
+  - /v2/docs/oryx
+  - /v2/docs/en/oryx
+  - /v1/docs/oryx
+  - /v1/docs/en/oryx
+related:
+  - title: Oryx for Zed
+    link: docs/scos/dev/front-end-development/zed/oryx-for-zed.html
+---
+
+{% info_block warningBox "No longer supported" %}
+
+This package is no longer supported. All functionality has been moved to [Oryx for Zed](/docs/scos/dev/front-end-development/zed/oryx-for-zed.html) starting from *2.13.0* version.
+
+{% endinfo_block %}
```

Comment:
In this file I've just added a notification section, the rest part copied from the previous version